.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: rust
-rust: nightly
+rust:
+  - nightly
+  - 1.28.0
```

Comment:
This specifically runs the tests on `1.28.0` and the latest nightly (whenever a test run is triggered). When the next stable is release you could add as a third version to run the tests against three versions, but since it's stable rust it should not break. Switching from `1.28.0` to `1.29.0` would kind of imply the minimum required version is `1.29.0` because the crate could use stabilizations that were not yet available in `1.28.0`.